Solutions/Armis/Data/Solution_Armis.json

```diff
@@ -0,0 +1,22 @@
+{
+    "Name": "Armis",
+    "Author": "{Armis - support@armis.com}",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/Armis.svg\" width=\"75px\" height=\"75px\">",
+    "Description": "The [Armis](https://www.armis.com/) Solution gives the capability to ingest Armis Devices, Alerts and device Activities into Microsoft Sentinel through the Armis REST API.",
+    "Parsers":[
+        "Parsers/ArmisActivities.txt",
+        "Parsers/ArmisDevice.txt",
+        "Parsers/ArmisAlerts.txt"
+    ],
+    "Data Connectors": [
+        "Data Connectors/ArmisActivities/ArmisActivities_API_FunctionApp.json",
+        "Data Connectors/ArmisDevice/ArmisDevice_API_FunctionApp.json",
+        "Data Connectors/ArmisAlerts/ArmisAlerts_API_FunctionApp.json"
+    ],
+    "Playbooks":[
+        "Playbooks/ArmisUpdateAlertStatus/azuredeploy.json"
+    ],
+    "BasePath": "C:\\Azure-Sentinel\\Solutions\\Armis",
+    "Version": "2.0.0",
+    "Metadata": "SolutionMetadata.json"
```

Comment:
Can you add "TemplateSpec": true, "Is1PConnector": false in input file? Please refer this [doc ](https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Create-Azure-Sentinel-Solution/V2)for reference, thanks.